lib/constants.go

```diff
@@ -707,8 +716,8 @@ type DeSoParams struct {
 	// The maximum aggregate number of bytes of transactions included in the PoS mempool.
```

Comment:
These fields were stored in flags. If that's the case they shouldn't be in constants.go. constants.go is for consensus-level constants, not flag constants.

Reply:
Oh, you're right!